src/import/load-costume.js

```diff
@@ -15,18 +15,31 @@ const log = require('../util/log');
  */
 const loadCostumeFromAsset = function (costume, costumeAsset, runtime) {
     costume.assetId = costumeAsset.assetId;
-    if (!runtime.renderer) {
+    const renderer = runtime.renderer;
+    if (!renderer) {
         log.error('No rendering module present; cannot load costume: ', costume.name);
         return costume;
     }
     const AssetType = runtime.storage.AssetType;
-    const rotationCenter = [
-        costume.rotationCenterX / costume.bitmapResolution,
-        costume.rotationCenterY / costume.bitmapResolution
-    ];
+    let rotationCenter;
+    if (costume.rotationCenterX && costume.rotationCenterY && costume.bitmapResolution) {
```

Comment:
Is there actually a way to get in here without a valid `bitmapResolution`? If so, see below; if not, I would remove it from this test to avoid confusion.

Reply:
It is possible, via file upload.